tests/test_adjoint_jacobian.py

```diff
@@ -288,7 +288,6 @@ def test_gradients(self, op, obs, tol, dev_cpu, dev_kokkos):
 
             qml.Rot(1.3, -2.3, 0.5, wires=[0])
             qml.RZ(-0.5, wires=0)
-            qml.RY(0.5, wires=1).inv()
```

Comment:
As mentioned in the other PR about replacing `inv`. Same for all tests that follow

Reply:
done!